### ISSUE TITLE
Uncaught TypeError: Cannot set property 'innerHTML' of undefined

### ISSUE THREAD

**Issue by BrennanAyers** [Fred-Barclay/Termination#96]
Enter steps to reproduce:

1. Be using Termination as normal
2. Have files to commit in Git
3. Have a prebuilt COMMIT_EDITMSG
4. Type `gc`, which opens `vi` to edit commit message
5. Throw Exception

This is the current commit message that I believe started these issues, as we had been using this workflow for the previous hours/days.
This exception only gets thrown in Termination, using normal Mac Terminal will not cause issues on loading `vi`.
```


Co-Authored-By: Me <my-email@email.com>
Co-Authored-By: Teammate1 <teammate1-email@email.com>
Co-Authored-By: Teammate2 <teammate2-email@email.com>
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch 7-email-activiation
# Changes to be committed:
#       modified:   app/views/verification_mailer/verification_email.html.erb
#       modified:   app/views/verification_mailer/verification_email.text.erb
#
# Changes not staged for commit:
#       modified:   app/controllers/users_controller.rb
#       modified:   config/routes.rb
#       modified:   spec/features/user/user_receives_activation_email_on_registration_spec.rb
#
# Untracked files:
#       app/controllers/email_verification_controller.rb
```

**Atom**: 1.38.2 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.5
**Thrown From**: [termination](https://github.com/Fred-Barclay/Termination) package 0.7.1

### Stack Trace

Uncaught TypeError: Cannot set property 'innerHTML' of undefined

```
At /Users/brennan/.atom/packages/termination/node_modules/term.js/src/term.js:1704

TypeError: Cannot set property 'innerHTML' of undefined
    at Terminal.refresh (/packages/termination/node_modules/term.js/src/term.js:1704:32)
    at Terminal.write (/packages/termination/node_modules/term.js/src/term.js:2897:8)
    at /packages/termination/lib/view.coffee:141:17
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:38293
    at Function.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1206583)
    at Emitter.emit (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1208024)
    at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:37394)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:772:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

### Commands

```
     -0:55.3.0 termination:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
atom-html-preview 0.2.6 
busy-signal 2.0.1 
erb-snippets 1.1.0 
file-icons 2.1.32 
intentions 1.1.5 
language-slim 1.0.0 
linter 2.3.0 
linter-ruby 1.3.1 
linter-slim 2.0.0 
linter-ui-default 1.7.1 
markdown-preview-plus 3.8.1 
minimap 4.29.9 
minimap-autohider 1.6.0 
rainbow-csv 0.3.0 
remove-whitespace 0.2.1 
teletype 0.13.4 
termination 0.7.1 
```

**Comment by Fred-Barclay** [comment]
Hi @BrennanAyers 
I'm sorry for the late reply. I wasn't able to find a way to debug this since I don't have access to macOS.

I've just pushed out a new release and Atom has been updated to 1.39. If you're able, can you let me know if this problem still persists with the newer versions?
Cheers!
Fred

**Comment by BrennanAyers** [comment]
Hey @Fred-Barclay 
No worries, I've been busy as well. It has also been good to get used to the Alt-Tab workflow again 😄 
I just made sure Atom was updated, and reinstalled the package. Everything seems to be working well so far, I was able to get the `gc` Vi instance going and nothing would crash. I also checked out the [Platform-IO-IDE Terminal](https://github.com/platformio/platformio-atom-ide-terminal) issues board, and there were a handful of people with similar issues. I'm thinking it was something unrelated to my commit message, and more to do with an Atom update.
I will close my other ticket as well, as that also cropped up in the other repo and seemed to be something to do with a change in Python libraries in Atom.

Thanks for all the work you do!
